package.json

```diff
@@ -58,8 +49,18 @@
     "@types/pg": "^8.11.6",
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
+    "@typescript-eslint/eslint-plugin": "^7.16.1",
+    "@typescript-eslint/parser": "^7.16.1",
+    "eslint": "^8.57.0",
+    "eslint-config-next": "^14.2.5",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-custom-rules": "file:./eslint-plugin-custom-rules",
+    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-react": "^7.34.4",
+    "git-commit-msg-linter": "^5.0.8",
```

Comment:
This PR uses `git-commit-msg-linter` instead of `commitlint`, which @nlebovits mentioned in the ticket. `commitlint` has a much larger community of contributors than `git-commit-msg-linter`, and it also has many more users (100 times more weekly downloads). What are the pros and cons of using `git-commit-msg-linter` instead of a library such as `commitlint`?